docs/general/fields.md

```diff
@@ -0,0 +1,23 @@
+[Back to Guides](../README.md)
+
+# Fields
+
+If for a reason, you need to override the fields defined on serializer, you should use `fields`
+
+For example, if you have a serializer like this
+
+```ruby
+class UserSerializer < ActiveModel::Serializer
+  attributes :access_token, :first_name, :last_name
+end
+```
+
+and in a specific controller, you can't return the `first_name`, `fields` will help you
```

Comment:
```diff
-and in a specific controller, you can't return the `first_name`, `fields` will help you
+and in a specific controller you want to return the `access token` only, the `fields` option will help you:
```

The `last_name` was missing here and better stay positive rather than negative :smile: